### ISSUE TITLE
non-reproducible error: 'ranged' should be a list of 4-element integer vectors

### ISSUE THREAD

**Issue by jdtrat** [MilesMcBain/datapasta#117]
Hello! 

This is probably a simple fix, but I can't figure it out. I get an error when running datapasta functions in my editor but not with a reprex. I'm assuming there might be a conflict with one of my other attached packages (`sessionInfo()` below), but would appreciate any thoughts!

## editor

\> library(datapasta)
\> library(magrittr)
\> letters %>%
\+   head() %>%
\+   vector_paste_vertical()
Error: 'ranges' should be a list of 4-element integer vectors
\> 
\> sessionInfo()
R version 4.0.2 (2020-06-22)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Catalina 10.15.7

Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] graphics  grDevices utils     datasets  stats     methods   base     

other attached packages:
[1] clipr_0.7.1     datapasta_3.1.0 ISLR_1.2        arsenal_3.5.0   readxl_1.3.1   
[6] cowplot_1.1.0   rmarkdown_2.5   magrittr_2.0.1 

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.5       pillar_1.4.7     cellranger_1.1.0 compiler_4.0.2  
 [5] tools_4.0.2      digest_0.6.27    evaluate_0.14    lifecycle_0.2.0 
 [9] tibble_3.0.4     gtable_0.3.0     pkgconfig_2.0.3  rlang_0.4.9     
[13] reprex_0.3.0     rstudioapi_0.13  yaml_2.2.1       xfun_0.19       
[17] dplyr_1.0.2      knitr_1.30       fs_1.5.0         hms_0.5.3       
[21] generics_0.1.0   vctrs_0.3.5      rsthemes_0.1.0   grid_4.0.2      
[25] tidyselect_1.1.0 glue_1.4.2       R6_2.5.0         processx_3.4.5  
[29] callr_3.5.1      whisker_0.4      ggplot2_3.3.2    purrr_0.3.4     
[33] readr_1.4.0      ps_1.4.0         scales_1.1.1     ellipsis_0.3.1  
[37] htmltools_0.5.0  colorspace_1.4-1 munsell_0.5.0    crayon_1.3.4    

## reprex 
``` r
library(datapasta)
library(magrittr)
letters %>%
  head() %>%
  vector_paste_vertical()
#> c("a",
#>   "b",
#>   "c",
#>   "d",
#>   "e",
#>   "f")

sessionInfo()
#> R version 4.0.2 (2020-06-22)
#> Platform: x86_64-apple-darwin17.0 (64-bit)
#> Running under: macOS Catalina 10.15.7
#> 
#> Matrix products: default
#> BLAS:   /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRblas.dylib
#> LAPACK: /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRlapack.dylib
#> 
#> locale:
#> [1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8
#> 
#> attached base packages:
#> [1] stats     graphics  grDevices utils     datasets  methods   base     
#> 
#> other attached packages:
#> [1] magrittr_2.0.1  datapasta_3.1.0
#> 
#> loaded via a namespace (and not attached):
#>  [1] crayon_1.3.4    digest_0.6.27   R6_2.5.0        lifecycle_0.2.0
#>  [5] evaluate_0.14   pillar_1.4.7    highr_0.8       rlang_0.4.9    
#>  [9] stringi_1.5.3   rstudioapi_0.13 ellipsis_0.3.1  vctrs_0.3.5    
#> [13] rmarkdown_2.5   tools_4.0.2     stringr_1.4.0   readr_1.4.0    
#> [17] hms_0.5.3       xfun_0.19       yaml_2.2.1      compiler_4.0.2 
#> [21] pkgconfig_2.0.3 htmltools_0.5.0 knitr_1.30      tibble_3.0.4
```

<sup>Created on 2020-12-03 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by MilesMcBain** [comment]
Thanks for this!

Were you running RStudio or VSCode?

**Comment by emilelatour** [comment]
I've been getting the same issue starting today when using RStudio

**Comment by MilesMcBain** [comment]
I can't reproduce this using the same datapasta and rstudioapi versions as
you.

I'm using the current RStudio desktop release. Are you using a daily build?
Could please let me know the version?

**Comment by emilelatour** [comment]
I thought that I have the most recent RStudio desktop release: Version 1.4.869.

I am on MacOS with most recent Catalina

**Comment by emilelatour** [comment]
I re-dowloaded the recent desktop version of RStudio (Version 1.3.1093) and things are now working as the should

**Comment by MilesMcBain** [comment]
Thanks 👍 So you had a daily or preview build. We know now there is something coming down the pipe that will break `datapasta` and I can report it to RStudio.

**Comment by jdtrat** [comment]
Sorry, I just saw this! I'm also running the preview build of RStudio. Thanks for your help (and for reporting it to RStudio)!

**Comment by MilesMcBain** [comment]
It looks like this is fixed in the latest RStudio daily: Version 1.4.1080

**Comment by jdtrat** [comment]
Fast fix! Thanks so much.
